test/test_linalg.py

```diff
@@ -42,16 +45,6 @@
 if TEST_SCIPY:
     import scipy
 
-def setLinalgBackendsToDefaultFinally(fn):
-    @wraps(fn)
```

Comment:
Since this decorator definition moved, the lint error about import statement for wraps is legit